peepholelib/classifier/classifier_base.py

```diff
@@ -32,23 +32,18 @@ def null_parser(**kwargs):
     
 class ClassifierBase: # quella buona
     def __init__(self, **kwargs):
-        '''
-        load = kwargs['load'] if 'load' in kwargs else False
-        if load:
-            self.nl_class = None
-            self.nl_model = None
-            
-            self.device = None
-            self.parser = None
-            self.parser_kwargs = {}
-        else:
-        '''
+        
         self.nl_class = kwargs['nl_classifier']
         self.nl_model = kwargs['nl_model']
+
+        self.path = kwargs['path']
+        # create folder
+        self.path.mkdir(parents=True, exist_ok=True)
```

Comment:
Folder should be created on `save()`, so we do not create an empty folder if we do not save